frontend/src/components/project/submit/SubmitCard.vue

```diff
@@ -3,10 +3,12 @@
         <v-card-item>
             <v-card-title>{{ $t("submit.submit_title") }}</v-card-title>
         </v-card-item>
-        <v-container class="card-container">
+        <h1 v-if="isLoading">Loading...</h1>
+        <h1 v-else-if="isError">Error</h1>
+        <v-container v-else class="card-container">
             <v-row>
                 <v-col>
-                    <ProjectMiniCard :projectId="projectId" />
+                    <ProjectMiniCard :project="project!" />
```

Comment:
```suggestion
                    <v-skeleton-loader :loading="isLoading" type="article">
                        <ProjectMiniCard :project="project!" />
                    </v-skeleton-loader>
```

Reply:
hoeft niet per se,  maar anders krijg je elke keer Loading te zien

Reply:
Ik was van plan dit in milestone 3 eens voor de gehele applicatie in orde te brengen dus had ik het snel zo even gedaan, maar kan er inderdaad nu ook zeker al in.